libafl/src/events/llmp/mgr.rs

```diff
@@ -619,14 +619,21 @@ where
             #[cfg(feature = "llmp_compression")]
             let compressed;
             #[cfg(feature = "llmp_compression")]
-            let event_bytes = if _flags & LLMP_FLAG_COMPRESSED == LLMP_FLAG_COMPRESSED {
+            let event_bytes = if flags & LLMP_FLAG_COMPRESSED == LLMP_FLAG_COMPRESSED {
                 compressed = self.compressor.decompress(msg)?;
                 &compressed
             } else {
                 msg
             };
             let event: Event<S::Input> = postcard::from_bytes(event_bytes)?;
             log::debug!("Received event in normal llmp {}", event.name_detailed());
+
+            // If the message comes from another machine, do not
+            // consider other events than new testcase.
+            if !event.is_new_testcase() && (flags & LLMP_FLAG_FROM_MM == LLMP_FLAG_FROM_MM) {
```

Comment:
Why? We can totally share metadata etc in the future

Reply:
Pretty sure this check belongs into handle_in_client, if it should exist et all(?)

Reply:
yes, we were trying to bugfix something quickly for an experiment, i was planning to clean-up this today.
I'll reopen a pr soon

Reply:
Why merge tests to main?

Reply:
yes it was not a good idea in the first place, i'll just revert this one in the next patch